control_node: {
  control_is_on: false,
  joint_angle_desired: [-0.155, 0.126, -1.638, 1.509, -1.418, 1.538, -1.40],
  cartesian_pose_desired: [0,0,0,0,0,0],
  joint_velocity_desired: [0,0,0,0,0,0,0],
  selected_filter_number: 3,
  Kd: [20],
  Dd: [1],
  Lowpass_coeff: 0.6,
  move2neutral: false,
  suppress_self_collision_detection: false,
  suppress_contact_collision_detection: false,
}

oscillation_guard: {
  window_length: 50,
  corner_frequency: 20,
  magnitude_limit: 40,
}

nullspace: {
  is_locked: false,
  jointspace_pose: [-0.155, 0.126, -1.638, 1.509, -1.418, 1.538, -1.40],
}
named_poses: {
  right: {
    poses: {
      neutral: [-0.155, 0.126, -1.638, 1.509, -1.418, 1.538, -1.40],
    }
  }
}
